### ISSUE TITLE
The split in the training process

### ISSUE THREAD

**Issue by scuwyh2000** [Zhicaiwww/Diff-Mix#2]
Hi, great work!

I wanna ask one question, for the conventional image classification task, do you use the validation set or directly train and get the best performance on the test set?

**Comment by Zhicaiwww** [comment]
There is typically only two splits for the datasets that we use, i.e., train and validation. We report the best results on the validation dataset in the training with the same hyperparameters
